plugins/chruby/chruby.plugin.zsh

```diff
@@ -16,14 +16,15 @@ _source-from-omz-settings() {
 }
 
 _source-from-homebrew() {
-  (( $+commands[brew] )) || return 1
```

Comment:
On Apple Silicon macs `brew` is now in `/opt/homebrew/bin` instead of `/usr/local/bin` and it's not in the `PATH` by default, so this command fails and `command not found: chruby` is shown, while `chruby` could be loaded from the new default brew prefix. And it makes more sense to check to see if the `brew` command is available only before shelling out to the command.